class Blort
!!!1113143.java!!!	maybeThrow(in x : int) : int
        if (x < 10) {
            throw new RuntimeException();
        }

        return x;
!!!1113271.java!!!	exTest1(in x : int) : int
        try {
            maybeThrow(x);
            return 1;
        } catch (RuntimeException ex) {
            return 2;
        }
!!!1113399.java!!!	exTest2(in x : int) : int
        try {
            x++;
            x = maybeThrow(x);
        } catch (RuntimeException ex) {
            return 1;
        }

        // Since the code in the try block can't possibly throw, there
        // should not be a catch in the final result.
        try {
            x++;
        } catch (RuntimeException ex) {
            return 2;
        }

        try {
            return maybeThrow(x);
        } catch (RuntimeException ex) {
            return 3;
        }
